crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -409,54 +431,78 @@ impl HolochainP2pActor {
         }
     }
 
-    /* -----------------
-     * saving so we can implement similiar stuff later
+    async fn get_peer_for_loc(
+        &self,
+        tag: &'static str,
+        space: &DynSpace,
+        loc: u32,
+    ) -> HolochainP2pResult<(AgentPubKey, Url)> {
+        let mut agent_list = space
+            .peer_store()
+            .get_near_location(loc, 1024)
```

Comment:
Can use `get_near_location(loc, 0)` and skip filtering on `storage_arc.contains(loc)` below?

Reply:
Pretty sure that is a straight-up database limit operation. If we put 0 there we will never get any rows back.

Reply:
Ahh right, I'm mixing up "how far away from the location" with "how many results to return"